### PR TITLE
Removing freemarker config vars

### PR DESCRIPTION
**Pull request by amontenegro**

### CODE REVIEW

**Comment by amontenegro** [comment]
[Create new endpoint for environment configuration info currently passed via model attributes](https://trello.com/c/djXum1WJ/5676-create-new-endpoint-for-environment-configuration-info-currently-passed-via-model-attributes)

File: orcid-web/src/main/webapp/static/javascript/ng1Orcid/app/modules/oauthAuthorization/oauthAuthorization.component.ts [hunk 1]
```diff
@@ -698,34 +721,26 @@ export class OauthAuthorizationComponent implements AfterViewInit, OnDestroy, On
         }
 
         //if oauth request load request info form
-        if(orcidVar.oauth2Screens || orcidVar.originalOauth2Process){
+        const urlParams = new URLSearchParams(window.location.search);
```
Comment (leomendoza123) [inline orcid-web/src/main/webapp/static/javascript/ng1Orcid/app/modules/oauthAuthorization/oauthAuthorization.component.ts]: Just for the records. 
Is regularly not recommended to access directly global objects like window. This restricts the app to not use optimizations like angular server-side rendering, more info: https://brianflove.com/2018/01/11/angular-window-provider/

File: orcid-web/src/main/webapp/static/javascript/script.js [hunk 2]
```diff
@@ -329,9 +359,12 @@ function logAjaxError(e){
     console.log("responseText: " + e.responseText);*/
 }
 
-function getBaseUri() {
-    return 'https:' == document.location.protocol ? orcidVar.baseUri
-            : orcidVar.baseUriHttp;
+function getBaseUri() {    
+    var uri = location.protocol + '//' + location.host
```
Comment (leomendoza123) [inline orcid-web/src/main/webapp/static/javascript/script.js]: Just for the record. 
This ideally will be moved to be provided on a service in the future. 

This also applies for most function on script.js

**Comment by amontenegro** [comment]
Hey @lizkrznarich this is ready to go!

Thanks

**Pull request merged**